Content.Client/UserInterface/ItemSlotButton.cs

```diff
@@ -56,6 +60,23 @@ public ItemSlotButton(Texture texture, Texture storageTexture)
 
             StorageButton.OnPressed += OnStorageButtonPressed;
 
+            Button.OnMouseEntered += _ =>
+            {
+                MouseIsHovering = true;
+            };
+            Button.OnMouseEntered += OnButtonHover;
+
+            Button.OnMouseExited += _ =>
+            {
+                MouseIsHovering = false;
+                if (EntityHover)
+                {
+                    EntityHover = false;
```

Comment:
Entities are not garbage collected so you'll need to make sure to `Delete()` the hover entity here.

Reply:
now calling delete on the owner of the sprite before settings to null